template/webpack.config.js

```diff
@@ -2,40 +2,46 @@ const path = require('path')
 const moduleConfig = require('./webpack-module.config')
 const buildPage = require('tram-build-page')
 
-const dehydratedIndex = buildPage({
-  appPath: './main.js',
-  route: '/no-js',
-  indexPath: './public/index.html'
-})
+module.exports = env => {
+  // build default page (without javascript)
+  const dehydratedIndex = buildPage({
+    appPath: './main.js',
+    route: '/',
+    indexPath: './public/index.html',
+    outPath: env === 'production' ? './dist/index.html' : undefined
```

Comment:
we now create this file as an artifact of running webpack, instead of having a separate build step in npm.